react/react_testing/introduction_to_react_testing.md

```diff
@@ -123,30 +123,13 @@ describe("App component", () => {
 });
 ```
 
-The tests speak for themselves. In the first test, we utilize snapshots to check whether all the nodes render as we expect them to. In the second test, we simulate a click event. Then we check if the heading changed. `toMatch` is one of the various assertions we could have made. 
+The tests speak for themselves. In the first test, we utilize snapshots to check whether all the nodes render as we expect them to. In the second test, we simulate a click event. Then we check if the heading changed. `toMatch` is one of the various assertions we could have made. Notice that the callback function for the second test is an async one, as we need this in order to await user.click().
```

Comment:
```suggestion
The tests speak for themselves. In the first test, we utilize snapshots to check whether all the nodes render as we expect them to. In the second test, we simulate a click event. Then we check if the heading changed. `toMatch` is one of the various assertions we could have made. Notice that the callback function for the second test is an async one, as we need this in order to `await user.click()`.
```

Could you add the backticks here since it's code that's being referenced?

Reply:
> Could you add the backticks here since it's code that's being referenced?

Okay, I've also added backticks to the `async` keyword.